### PR TITLE
chore(release): add release notes for operator and halyard

### PR DESCRIPTION
**Pull request by cristhian-castaneda**

### CODE REVIEW

File: content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: v1.12.0 Armory Halyard
+toc_hide: true
+version: 01.12.00
+description: Release notes for Armory Halyard v1.12.0
+---
+
+## 05/07/2021 Release Notes
```
Comment (dogonthehorizon) [inline content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md]: We should ask @jorgebee65 and @rvazquezglez for something we can use to highlight the fact that Docker registries can use regex filters now

File: content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-6.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: v1.2.6 Armory Operator
+toc_hide: true
+version: 01.02.06
+description: Release notes for Armory Operator v1.2.6
+---
+
+## 05/07/2021 Release Notes
```
Comment (dogonthehorizon) [inline content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-6.md]: And probably include that note about registries in this release as well?

File: content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md (hunk 1 above)
Comment (dogonthehorizon) [reply content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md]: Or maybe @brian-armory already has a snippet for this, I don't remember

Comment (317brian) [reply content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md]: taken care of

File: content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-6.md (hunk 2 above)
Comment (317brian) [reply content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-6.md]: taken care of

**Review by dogonthehorizon — APPROVED** [review]

**Pull request merged**